src/main/java/com/neprozorro/model/LotInfo.java

```diff
@@ -0,0 +1,63 @@
+package com.neprozorro.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "lot_info")
+public class LotInfo {
+
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @Column(name = "id")
+    private UUID id;
+
+    @ManyToOne
+    private Participant buyer;
+
+    @ManyToOne
```

Comment:
consider making it Lazy, by default toOne is Eager

Reply:
Done